examples/compose/plot_column_transformer_mixed_types.py

```diff
@@ -37,9 +37,9 @@
 np.random.seed(0)
 
 # Read data from Titanic dataset.
-titanic_url = ('https://raw.githubusercontent.com/amueller/'
-               'scipy-2017-sklearn/091d371/notebooks/datasets/titanic3.csv')
-data = pd.read_csv(titanic_url)
+titantic = fetch_openml(data_id=40945, as_frame=True)
```

Comment:
Do we want to use return_X_y here also?

Reply:
I usually prefer to show case the Bunch object.